Cargo.toml

```diff
@@ -92,6 +68,7 @@ cargo-generate = "0.22"
 wasm-opt = "0.116.1"
 ignore = "0.4.23"
 walkdir = "2.5"
+openssl = { version = "0.10.71", features = ["vendored"] }
```

Comment:
instead of adding the dep here, I think the `cargo-generate` dep has a feature flag for vendored openssl and git2 which maybe a better fix?

Reply:
I've confirmed this fixes the issue for me on Alpine (musl) builds for `aarch64-unknown-linux-musl`.

I had to add a few more deps to my docker image:

- make
- perl

Reply:
I've added https://github.com/leptos-rs/cargo-leptos/pull/452 for reference

Reply:
We already added `libssl-dev` as part of build process. Please remove this line too.